include/graphit/frontend/fir.h

```diff
@@ -1673,6 +1673,21 @@ namespace graphit {
                virtual void copy(FIRNode::Ptr);
         };
 
+        struct ConstantVectorExpr : public Expr {
```

Comment:
I feel like we don't need to add this ConstantVectorExpr. We should be able to get the size information from the VectorTyp. We can talk more during the meetings.